imports/ui/pages/Community.js

```diff
@@ -1,15 +1,12 @@
 import React, { Component, useState } from "react";
 import { Meteor } from "meteor/meteor";
 import { withTracker } from "meteor/react-meteor-data";
-import { Logger } from "../../../lib/client/Logger";
 import AppWrapper from "../pages/components/AppWrapper";
 
 import Messenger from "./components/Chat/Messenger";
 import { Button, Input, Modal } from "antd";
 import { Chat, Rooms, mongoCss } from "../../api/client/collections";
-
-// eslint-disable-next-line no-unused-vars
-const log = new Logger("client/Community");
+import { translate } from "../HOCs/translate";
```

Comment:
HOCs naming convention is: `with<HocName>`. Let's rename it to `withTranslate`.
I see no sense have in that file more that one HOC - let's export ``withTranslate` by default (`export default translate = namespace => WrappedComponent => { ... `) and import: `import withTranslate from "../HOCs/withTranslate";`